src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Debug/StartupProjectRegistrar.cs

```diff
@@ -66,10 +65,8 @@ protected override async Task InitializeCoreAsync(CancellationToken cancellation
 
             Assumes.Present(_startupProjectsListService);
 
-            _subscription = _projectSubscriptionService.ProjectRuleSource.SourceBlock.LinkTo(
-                target: new ActionBlock<IProjectVersionedValue<IProjectSubscriptionUpdate>>(OnProjectChangedAsync),
-                suppressVersionOnlyUpdates: true,
```

Comment:
Just checking, why these two lines are not needed anymore?

Reply:
supressVersionOnlyUpdates and DataflowOption.PropagateCompletion are the defaults here.

Reply:
ah ok :)